lib/action_view/helpers/dynamic_form.rb

```diff
@@ -11,14 +11,18 @@ module Helpers
     # is a great way of making the record quickly available for editing, but likely to prove lackluster for a complicated real-world form.
     # In that case, it's better to use the +input+ method and the specialized +form+ methods in link:classes/ActionView/Helpers/FormHelper.html
     module DynamicForm
+      DEFAULT_FIELD_OPTION_MAXLENGTH = 30
       # Returns a default input tag for the type of object returned by the method. For example, if <tt>@post</tt>
       # has an attribute +title+ mapped to a +VARCHAR+ column that holds "Hello World":
       #
       #   input("post", "title")
-      #   # => <input id="post_title" name="post[title]" size="30" type="text" value="Hello World" />
-      def input(record_name, method, options = {})
-        raise_broken_code_error
-        InstanceTag.new(record_name, method, self).to_tag(options)
+      #   # => <input id="post_title" name="post[title]" size="30" maxlength="30" type="text" value="Hello World" />
```

Comment:
I find more value in adding the maxlength attribute as well, in addition to the size.